network/endpoint_linux.go

```diff
@@ -1,8 +1,6 @@
 // Copyright 2017 Microsoft. All rights reserved.
 // MIT License
 
-// +build linux
```

Comment:
Should we rename? `endpoint_linux.go` specifically implies to me "// +build linux" by convention.

Reply:
Oh you are just doing this because its already included by default via the naming?

Reply:
I was just being consistent with what's there.